File: src/Entity/Commentaire.php
<?php

namespace App\Entity;

use App\Repository\CommentaireRepository;
use Doctrine\ORM\Mapping as ORM;

#[ORM\Entity(repositoryClass: CommentaireRepository::class)]
class Commentaire
{
    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column]
    private ?int $id = null;

    //#[ORM\ManyToOne]
    //#[ORM\JoinColumn(nullable: false)]
    //private ?Cours $id_cours = null;

    #[ORM\Column(length: 255)]
    private ?string $contenu = null;

    #[ORM\Column]
    private ?int $note = null;
    //
    // #[ORM\ManyToOne]
    // #[ORM\JoinColumn(nullable: false)]
    // private ?User $id_user = null;

    public function getId(): ?int
    {
        return $this->id;
    }

    // public function getIdCours(): ?Cours
    // {
    //     return $this->id_cours;
    // }

    // public function setIdCours(?Cours $id_cours): self
    // {
    //     $this->id_cours = $id_cours;
    //
    //     return $this;
    // }

    public function getContenu(): ?string
    {
        return $this->contenu;
    }

    public function setContenu(string $contenu): self
    {
        $this->contenu = $contenu;

        return $this;
    }

    public function getNote(): ?int
    {
        return $this->note;
    }

    public function setNote(int $note): self
    {
        $this->note = $note;

        return $this;
    }

    //public function getIdUser(): ?User
    //{
    //    return $this->id_user;
    //}
    //
    //public function setIdUser(?User $id_user): self
    //{
    //    $this->id_user = $id_user;
    //
    //    return $this;
    //}
}

File: pimomo/src/main/java/com/example/pimomo/Commentaire.java
package com.example.pimomo;

/**
 *
 * @author [me]
 */
public class Commentaire {
    
    private int id,id_cours;
    private String contenu;
    private int note;
    private Cours cours;

    public Commentaire(String contenu, int note, int coursId) {
        this.contenu = contenu;
        this.note = note;
        this.id_cours = coursId;
    }

    public void setCours(Cours cours) {
        this.cours = cours;
    }

    public int getId_cours() {
        return id_cours;
    }

    public void setId_cours(int id_cours) {
        this.id_cours = id_cours;
    }

    public Commentaire() {
    }

    public Commentaire(int id, String contenu, int note,int id_cours) {
        this.id = id;

        this.contenu = contenu;
        this.note = note;
        this.id_cours=id_cours;
    }

    public Commentaire(String contenu, int note) {
        this.contenu = contenu;
        this.note = note;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getContenu() {
        return contenu;
    }

    public void setContenu(String contenu) {
        this.contenu = contenu;
    }

    public int getNote() {
        return note;
    }

    public void setNote(int note) {
        this.note = note;
    }

    @Override
    public String toString() {
        return "Commentaire{" + "id=" + id + ", contenu=" + contenu + ", note=" + note + '}';
    }
    
}
